src/pages/Integrations/Create/schema.ts

```diff
@@ -1,45 +1,44 @@
 import { componentTypes } from '@data-driven-forms/react-form-renderer';
-
-export const schema = {
-  fields: [
-    {
-      component: componentTypes.WIZARD,
-      inModal: true,
-      title: 'Add integration',
-      description:
-        'Configure integrations between third-party tools and the Red Hat Hybrid Cloud Console.',
-      name: 'add-integration-wizard',
-      fields: [
+import { integrationsStep } from './IntegrationsStep';
+export const schema = (category) => ({
+    fields: [
         {
-          title: 'Enter details',
-          name: 'details',
-          nextStep: 'review',
-          fields: [
-            {
-              component: componentTypes.TEXT_FIELD,
-              name: 'integration-name',
-              type: 'text',
-              label: 'Integration name',
-              isRequired: true,
-              validate: [
+            component: componentTypes.WIZARD,
+            inModal: true,
```

Comment:
you can just add `className: 'sources',` and use the cardSelect.scss from sources (with some cleanup) and the styling should be fixed